packages/asana/plugin-docs.yaml

```
@@ -0,0 +1,12 @@
+displayName: Asana
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)